applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -200,6 +214,10 @@ def _GetOrderOfProcessesInitialization(self):
     def _GetSimulationName(self):
         return "GeoMechanics Analysis"
 
+    def _GetMinDeltaTimeValueOrDefault(self):
+        delta_time_as_fraction_of_time_span = 0.001
+        return self.min_delta_time if self.min_delta_time is not None else min(self.initial_delta_time,delta_time_as_fraction_of_time_span*(self.end_time - self.start_time))
```

Comment:
Adding a few spaces improves readability:
```suggestion
        return self.min_delta_time if self.min_delta_time is not None else min(self.initial_delta_time, delta_time_as_fraction_of_time_span * (self.end_time - self.start_time))
```

Reply:
added spaces